apps/heureka/src/hooks/useCommunication.js

```diff
@@ -3,65 +3,103 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { useEffect } from "react"
-import { broadcast, get, watch } from "@cloudoperators/juno-communicator"
+import { useEffect, useCallback } from "react"
+import { get, watch } from "@cloudoperators/juno-communicator"
 import {
   useUserActivityActions,
-  useAuthAppLoaded,
-  useAuthIsProcessing,
-  useAuthError,
-  useAuthLoggedIn,
-  useAuthLastAction,
   useAuthActions,
+  useIssueMatchesActiveFilters,
+  useServiceActiveFilters,
+  useComponentActiveFilters,
+  useFilterActions,
+  useIssueMatchesFilterLabels,
+  useServiceFilterLabels,
+  useComponentFilterLabels,
 } from "./useAppStore"
-import { AUTH_ACTIONS } from "../lib/slices/createAuthDataSlice"
 
 const useCommunication = () => {
+  console.debug("[heureka] useCommunication setup")
   const { setIsActive } = useUserActivityActions()
-  const authAppLoaded = useAuthAppLoaded()
-  const authIsProcessing = useAuthIsProcessing()
-  const authError = useAuthError()
-  const authLoggedIn = useAuthLoggedIn()
-  const authLastAction = useAuthLastAction()
-  const { setData: authSetData, setAppLoaded: authSetAppLoaded } = useAuthActions()
+  const { setData: authSetData } = useAuthActions()
+
+  // Active filters for each entity
+  const issueMatchesActiveFilters = useIssueMatchesActiveFilters()
+  const serviceActiveFilters = useServiceActiveFilters()
+  const componentActiveFilters = useComponentActiveFilters()
+
+  // Filter labels for each entity
+  const issueMatchesFilterLabels = useIssueMatchesFilterLabels()
+  const serviceFilterLabels = useServiceFilterLabels()
+  const componentFilterLabels = useComponentFilterLabels()
+
+  const { setActiveFilters } = useFilterActions()
+
+  const setAuthData = useCallback(
```

Comment:
should be useCallback here and why?

Reply:
I removed the section for adding a default filter based on the logged-in user's support group. Since authentication doesn't occur in the Heureka API and we currently lack a solid solution across Greenhouse apps to utilize its auth info, the feature isn't feasible yet. I'll implement it later.